kuma/javascript/src/account/routes.jsx

```diff
@@ -0,0 +1,65 @@
+// @flow
+import * as React from 'react';
+import Route from '../route.js';
+import Page from '../base/page.jsx';
+import LandingPage from './pages/index.jsx';
+
+type AccountRoutesParams = {
```

Comment:
What do you think about abstracting `AccountRoutesParams` and `AccountPageProps` to a generic type that can be shared with the `/payments` pages?